test/Spec.hs

```diff
@@ -173,6 +175,12 @@ main =
               it "pinned" $ do
                 ruleCatchesNot gemVersionPinned "RUN gem install bundler:1"
                 onBuildRuleCatchesNot gemVersionPinned "RUN gem install bundler:1"
+              it "does not warn on -v" $ do
+                ruleCatchesNot gemVersionPinned "RUN gem install bundler -v '2.0.1'"
+                onBuildRuleCatchesNot gemVersionPinned "RUN gem install bundler -v '2.0.1'"
```

Comment:
`gem` also support `--version` as `-v`. Open new issue?

Reply:
First confirm that you are using the latest version. I can't reproduce your case locally.

Reply:
```shell
$ hadolint --version
Haskell Dockerfile Linter 1.17.1-no-git
$ hadolint Dockerfile 
Dockerfile:5 DL3008 Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`
Dockerfile:5 DL3009 Delete the apt-get lists after installing something
Dockerfile:15 DL3008 Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`
Dockerfile:15 DL3009 Delete the apt-get lists after installing something
Dockerfile:34 DL3028 Pin versions in gem install. Instead of `gem install <gem>` use `gem install <gem>:<version>`
Dockerfile:66 DL3025 Use arguments JSON notation for CMD and ENTRYPOINT arguments
$
```

Link to Dockerfile: https://github.com/evemonk/evemonk/blob/master/Dockerfile

```shell
$ cat Dockerfile | grep "gem install"
RUN gem install bundler --version "$BUNDLER_VERSION" --force
$ 
```

I will open new issue.